app/uk/gov/hmrc/tariffclassificationfrontend/views/forms/components/input_radiogroup.scala.html

```diff
@@ -22,13 +22,14 @@
         inputClass: Option[String] = None,
         hint: Option[String] = None,
         labelClass: Option[String] = None,
+        heading:String = "Review status",
```

Comment:
I agree to move the heading to the top, but default it to Review status sounds the object will be less generic.